classes/log/real_time_logger.php

```diff
@@ -90,4 +90,13 @@ public function log_object_query($queryname, $objectcount, $objectsum = 0) {
         error_log($logstring);
         // @codingStandardsIgnoreEnd
     }
+
+    public function log_lock_timing($lock) {
```

Comment:
this function is duplicated, can it be shared in a base class somewhere?

Reply:
Refactored in next commit. 
log_lock_timing method moved to objectfs_logger class and extended in null_logger class as a dummy.